lib/measures/ReportRetrofitEmbodiedCarbon/measure.py

```diff
@@ -45,6 +49,27 @@ def modify_optimization_sheet(replacement_value):
     # Save updated workbook
     wb.save(new_optimization_excel_output_path)
 
+def optimization(self):
```

Comment:
@Swara-96 Your notebook has now been integrated here. Please familiarize and feel free to make changes here moving forward.